src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: 10px;
+  width: $calendar-width;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid #000;
+    background-color: $day-color;
```

Comment:
The variable `$day-color` is used here but is not defined within this file. Ensure that it is defined or imported from another file to avoid compilation errors.